tests/integration_tests/config/sim_showers_for_trigger_rates_run.yml

```diff
@@ -7,8 +7,10 @@ CTA_SIMPIPE:
     PRIMARY: proton
```

Comment:
Should we also test for `site: North`?

Reply:
@VictorBarbosaMartins - I guess yes, or?

Reply:
yes, I am going to include it now

Reply:
Pronto. I included one test for the south as well.